packages/react-web/src/__helpers__/jest.setup.ts

```diff
@@ -0,0 +1,18 @@
+import { Blob } from 'buffer';
+import crypto from 'crypto';
+import { TextEncoder, TextDecoder } from 'util';
+
+Object.defineProperty(global, 'crypto', {
+  // eslint-disable-next-line @typescript-eslint/no-unsafe-assignment
+  value: Object.setPrototypeOf({ subtle: crypto.subtle }, crypto),
+});
+
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+global.Blob = Blob;
+
+// until https://github.com/jsdom/jsdom/issues/2524 is resolved
+global.TextEncoder = TextEncoder;
+// eslint-disable-next-line @typescript-eslint/ban-ts-comment
+// @ts-ignore
+global.TextDecoder = TextDecoder;
```

Comment:
This content is familiar... is this needed by XMTP SDK?

Reply:
I've just checked, and no, it's because in react-web tests I included import of `@lens-protocol/react` package and error comes somewhere from lit package. we have the same jest.setup in react package

<img width="1048" alt="Screenshot 2023-07-24 at 11 03 19" src="https://github.com/lens-protocol/lens-sdk/assets/605420/30eed63c-05d3-486f-bc26-47a1c43da550">

Reply:
got it